src/components/modal/Modal.scss

```diff
@@ -0,0 +1,184 @@
+@import "../../partials/mixins";
+@import "../../partials/variables";
+
+
+.overlay {
+    width: 100vw;
+    height: 100vh;
+    top: 0;
+    left: 0;
+    right: 0;
+    bottom: 0;
+    position: fixed;
+}
+
+.overlay {
+    // background: rgba(49, 49, 49, 0.8);
```

Comment:
Try to not have commented code

Reply:
Yes correct, not the final need to rem these px's.